### PR TITLE
update the bug report template

### PR DESCRIPTION
**Pull request by drochetti**
updated following the template used by the iOS team.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -2,20 +2,52 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'Bug'
+labels: ''
```
Comment (richardmcclellan) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: :+1:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -2,20 +2,52 @@
...
+ - Include relevant Android log output (see https://github.com/aws-amplify/amplify-android/blob/main/CONTRIBUTING.md#getting-more-output)
+ - Include any relevant CLI log output under `~/.amplify/logs/amplify-cli-<issue-date>.log`
+
+**Is this a regression (i.e. was this working before a version upgrade?)
+*If applicable provide the versions used before and after the upgrade.*
+
+**Device Information (please complete the following information):**
+ - Device: [e.g. Pixel 4a, Emulator]
+ - Android Version: [e.g. 9, 10]
+ - Specific to Emulators:
```
Comment (richardmcclellan) [inline .github/ISSUE_TEMPLATE/bug_report.md:47]: What does this line mean?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -2,20 +2,52 @@
...
 
-1. A **short** description
-2. The version of Amplify Android you're using
-3. The **code** you're using to call Amplify Android
-4. Any relevant [logs](https://github.com/aws-amplify/amplify-android/blob/main/CONTRIBUTING.md#getting-more-output)
-5. Any relevant guides or documentation you're referencing
-6. For API or DataStore issues, your `schema.graphql`
-7. For CLI issues, include any relevant logs in `~/.amplify/logs/amplify-cli-<date>.log`
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (richardmcclellan) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
_A clear and concise description of what the bug is._
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -2,20 +2,52 @@
...
 
+**Expected behavior**
+_Replace this with a clear and concise description of what you expected to happen._
+
+**Screenshots**
+_If applicable, add screenshots to help explain your problem._
+
+**Environment(please complete the following information):**
+ - Amplify Framework Version: [e.g. 1.17.3]
+ - Amplify Categories:
```
Comment (richardmcclellan) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: This seems redundant with the labels that we add to issues.   Usually it's pretty obvious from the description which category they are asking about.

**Review by richardmcclellan — COMMENTED** [review]
This template would be great for an end user reporting a bug while using a mobile app with a UI.  Since Amplify is a library though, I think it should be a bit closer to what we have currently.  Instead of steps to reproduce, we should ask for a code sample.  Instead of screenshots, we should ask for log output (I know you still have it, but now it's kinda buried under environment).

I do think the formatting changes make it easier to fill out - that is, having a bold title, and italicizing everything that should be replaced.

I'd also prefer that we only the critical questions, otherwise the issue reports get so long that it becomes burdensome to read them.  Device information doesn't seem super relevant.  There are times when it's useful, but it's less than 1% of the time, and when we need it, we can ask for it.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -2,20 +2,52 @@
...
+   - [ ] API
+   - [ ] Auth
+   - [ ] DataStore
+   - [ ] Predictions
+   - [ ] Storage
+ - CLI Version: [e.g. 4.41.2, Run `amplify version`]
+ - Include relevant Android log output (see https://github.com/aws-amplify/amplify-android/blob/main/CONTRIBUTING.md#getting-more-output)
+ - Include any relevant CLI log output under `~/.amplify/logs/amplify-cli-<issue-date>.log`
+
+**Is this a regression (i.e. was this working before a version upgrade?)
```
Comment (changxu0306) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]:
```suggestion
**Is this a regression? (i.e. Was this working before a version upgrade?)
```

**Comment by richardmcclellan** [comment]
Closing in favor of https://github.com/aws-amplify/amplify-android/pull/1338

**Pull request closed without merging**